### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by sajid-munawar**
Screenshot from 2022-08-26 09-54-45
Screenshot from 2022-08-26 09-55-55
Screenshot from 2022-08-26 09-56-13
Screenshot from 2022-08-26 09-56-31
Screenshot from 2022-08-26 09-56-43
Screenshot from 2022-08-26 09-56-53
Screenshot from 2022-08-26 09-57-06
Screenshot from 2022-08-26 09-57-17

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -7,3 +7,24 @@ INSERT INTO ANIMALS VALUES(2,'Gabumon','Nov 15, 2018',2,true,8);
...
+SET DATE_OF_BIRTH = 'JUN 12, 2005'
+WHERE ID = 8;
+
+update animals
+set species = 'digimon'
+where name like '%mon';
+
+update animals
+set species = 'pokemon'
+where not species = 'digimon';
```
Comment (vechicin) [inline data.sql]: According to the project requirements, this query should be added to the `queries.sql` file. Please fix this.

File: queries.sql [hunk 2]
```diff
@@ -1,34 +1,66 @@
 /*Queries that provide answers to the questions from all projects.*/
```
Comment (vechicin) [inline queries.sql:1]: According to the project requirements, the following query should be present. Please add it.

Screen Shot 2022-08-26 at 8 52 05

File: queries.sql [hunk 3]
```diff
@@ -1,34 +1,66 @@
...
```
Comment (vechicin) [inline queries.sql:2]: According to the project requirements, the following query should be present, please add it.

Screen Shot 2022-08-26 at 8 53 39

File: queries.sql [hunk 4]
```diff
@@ -1,34 +1,66 @@
...
 
-
```
Comment (vechicin) [inline queries.sql:3]: According to the project requirements, the following query should be present, please add it.

Screen Shot 2022-08-26 at 8 55 49

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @sajid-munawar ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job updating the table and making most of the queries necessary required in this milestone.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @sajid-munawar ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**